### DISCUSSION TITLE
Include order

### DISCUSSION THREAD

**Discussion by xiaosuyang** [stack-of-tasks/pinocchio#2221] [Q&A] [answered]
Hello. When I make, a mistake "You should include pinocchio before the Boost headers (e.g. #include <pinocchio/fwd.hpp>)" occured,it seems that the include order had some problem, what can I do to fix it

**Comment by jcarpent** [comment, marked as answer]
Juste include `#include <pinocchio/fwd.hpp>` in the top of your hierarchy of header files.
